### DISCUSSION TITLE
Adaptive card css styling

### DISCUSSION THREAD

**Discussion by nda1974** [microsoft-search/pnp-modern-search#3370] [General]
I'm working on my first Serach Result webpart custom layout in the Modern Search V4 solution. Using the Adaptive Card layout file(custom-layout.json), I'm missing the opportunity to style the elements using css. Is there any to deploy or 'inject' css into the webpart?

**Comment by gretchunkim** [comment]
No. But you can do that using search extensibility.

**Reply by nda1974** [reply]
Thank you for your answer.
I am using search extensibility but just cant find a way to use css styling when customizing adaptive card view. Can you lead in the right direction please?

**Reply by gretchunkim** [reply]
I see two separate sources.   If I were you, use existing sample layouts go work around it for the shortcut method. 

- https://stackoverflow.com/questions/58105510/how-to-use-css-styling-on-adaptive-cards 
- https://learn.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/javascript/native-styling
